### PR TITLE
Update CONTRIBUTING.md about Homebrew/cask-versions repo

### PR DESCRIPTION
**Pull request by Goooler**
Homebrew/cask-versions has been deprecated and merged into this repo, so we should correct the message here.

--------------------

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Review by bevanjkay — COMMENTED** [review]
I think just saying "make sure it's not already in `homebrew-core`" makes more sense. As the question already implies that the application isn't in `homebrew-cask`

**Comment by Goooler** [comment]
That's a little implicit, users may miss the check in this repo. Either way, I can accept your suggestion if you want.

**Comment by bevanjkay** [comment]
It already made no mention of checking homebrew-cask, so I don't think we need to add it

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Follow the guide in [How to Open a Homebrew Pull Request](https://docs.brew.sh/H
 
 ## Adding a Cask
 
-Notice an application that's not in Homebrew Cask yet? Make sure it's not yet in [homebrew/cask-versions](https://github.com/Homebrew/homebrew-cask-versions) or [homebrew/core](https://github.com/Homebrew/homebrew-core) (can be searched with `brew search`). Mac App Store apps can't be installed via Homebrew Cask, but check out [mas](https://github.com/mas-cli/mas) for an alternative.
+Notice an application that's not in Homebrew Cask yet? Make sure it's not yet in this repo or [homebrew/core](https://github.com/Homebrew/homebrew-core) (can be searched with `brew search`). Mac App Store apps can't be installed via Homebrew Cask, but check out [mas](https://github.com/mas-cli/mas) for an alternative.
```
Comment (bevanjkay) [inline CONTRIBUTING.md:27]:
```suggestion
Notice an application that's not in Homebrew Cask yet? Make sure it's also not in [homebrew/core](https://github.com/Homebrew/homebrew-core) (can be searched with `brew search`). Mac App Store apps can't be installed via Homebrew Cask, but check out [mas](https://github.com/mas-cli/mas) for an alternative.
```

**Review by bevanjkay — APPROVED** [review]
Thank you @Goooler

**Pull request merged**